cmd/vic-machine/common/ops_credentials.go

```diff
@@ -47,6 +48,12 @@ func (o *OpsCredentials) Flags(hidden bool) []cli.Flag {
 			Usage:  "Password or token for the operations user. Defaults to the credential supplied with target",
 			Hidden: hidden,
 		},
+		cli.BoolFlag{
+			Name:        "ops-init-perms",
+			Usage:       "Initialize Roles and Permissions for ops-user",
```

Comment:
`Initialize roles and permissions for ops-user` would be more consistent with the other usage texts above, what do you think?